domains/0xmaruf.json

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "Personal Portfolion Hosted on Netlify ",
+    "repo": "https://github.com/Marufms4/0xMarufNextjs",
+    "owner": {
+        "username": "marufms4",
+        "email": "marufsm4@gmail.com"
+    },
+    "record": {
+        "A": ["75.2.60.5"]
```

Comment:
404 Site. Please send over a preview. Reopen this PR when you have this resolved.